support/fixtures/factories/assetAttribute.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('assetAttribute')
+  .attrs({
```

Comment:
Need the option here: `.option('fromServer', false)`

Reply:
🤦‍♂️ 

Added